src/configurations/adknowledgeportal/synapseConfigs/experimental_tools.ts

```diff
@@ -61,47 +61,49 @@ export const experimentalToolsCardConfiguration: CardConfiguration = {
   ],
 }
 
+export const experimentalDetailsTableConfigurationColumnLinks: LabelLinkConfig = [
```

Comment:
Refactored for reuse in the Project details page (where it shows the associated experimental models table)